client/gulpfile.js

```diff
@@ -67,6 +103,7 @@ gulp.task('watch', () => {
   })
   gulp.watch(sources.css.all, ['styles', 'stylelint'])
   gulp.watch(sources.scripts.all, ['scripts'])
+  gulp.watch(sources.images.svg.all, ['svgstore'])
```

Comment:
Agreed completly - I just didn't see the reason for the sub-typing, and just leaving them as `svg`s or even more specific `icons` - hierarchies in types seem to always create confusion and potentially complexity in builds etc so it can be good to just be explicit unless there's a good reason to have the relationship between images, svgs, and icons, which I can't see as the build steps will be completely different.